tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2978,49 +2980,26 @@ Status ConvertStridedSlice(OpConverterParams* params) {
           node_def.name());
     }
   }
+
```

Comment:
the is a comment for the old code, line 2972..2974 is the same as
const bool end_is_modified =
        !(end_mask & 1) && (!batch_size_is_defined || end[0] != input_dims[0]);